README.md

```diff
@@ -21,290 +44,131 @@ Building geth requires both a Go (version 1.7 or later) and a C compiler.
-   which uses different P2P bootnodes, different network IDs and genesis states.
-
-*Note: Although there are some internal protective measures to prevent transactions from crossing
-over between the main network and test network, you should make sure to always use separate accounts
-for play-money and real-money. Unless you manually move accounts, Geth will by default correctly
-separate the two networks and will not make any accounts available between them.*
-
-### Full node on the Rinkeby test network
-
-The above test network is a cross client one based on the ethash proof-of-work consensus algorithm. As such, it has certain extra overhead and is more susceptible to reorganization attacks due to the network's low difficulty / security. Go Ethereum also supports connecting to a proof-of-authority based test network called [*Rinkeby*](https://www.rinkeby.io) (operated by members of the community). This network is lighter, more secure, but is only supported by go-ethereum.
+Then, the geth console can start up and you can start a miner as follows:
 
 ```
-$ geth --rinkeby console
+> personal.newAccount()
+> miner.setEtherbase(eth.accounts[0])
+> miner.start()
 ```
 
-### Configuration
-
-As an alternative to passing the numerous flags to the `geth` binary, you can also pass a configuration file via:
+Then, once you are satisfied with mining for a few seconds, stop the miner with
 
 ```
-$ geth --config /path/to/your_config.toml
+> miner.stop()
 ```
 
-To get an idea how the file should look like you can use the `dumpconfig` subcommand to export your existing configuration:
+Now, save the passphrase you used in the geth node into a text file called password.txt. Then, once you have this private geth node running on your local network, we will need to generate fake, pending transactions that can then be processed into shards by validators and proposers. For this, we have created an in-house transaction generator CLI tool.
 
-```
-$ geth --your-favourite-flags dumpconfig
-```
+Transaction Generator
+---------------------
 
-*Note: This works only with geth v1.6.0 and above.*
+Work in Progress. To track our current draft of the tx generator cli spec, visit this [link](https://docs.google.com/document/d/1YohsW4R9dIRo0u5RqfNOYjCkYKVCmzjgoBDBYDdu5m0/edit?usp=drive_web&ouid=105756662967435769870).
 
-#### Docker quick start
+Once we have fake transactions broadcast to our local node, we can start a validator and proposer client in separate terminal windows to begin the sharding process.
 
-One of the quickest ways to get Ethereum up and running on your machine is by using Docker:
+Becoming a Validator
+-----------------------
+To deposit ETH and join as a validator in the Validator Manager Contract, run the following command:
 
 ```
-docker run -d --name ethereum-node -v /Users/alice/ethereum:/root \
-           -p 8545:8545 -p 30303:30303 \
-           ethereum/client-go
+geth validator --deposit 100eth --password ~/Desktop/password.txt
```

Comment:
Shouldn't there be the sharding client entry point ? like 
` geth shard validator --deposit 100eth --password ~/Desktop/password.txt`